catalog/Maintenance_Monitoring/Health_Check.yml

```diff
@@ -0,0 +1,12 @@
+name: Health Check
```

Comment:
We should go with plural "Health Checks" here please, i.e. we have "Web App Frameworks"; "JSON Parsers" and so on, and I suppose the gems themselves might also be capable of doing multiple checks depending on the use case so plural makes sense there as well